packages/homepage/src/components/Navigation/index.js

```diff
@@ -191,7 +191,9 @@ const Navigation = () => {
                   width: 100%;
                   background: #151515;
                   overflow: hidden;
-                  border-bottom: 1px solid ${props => props.theme.homepage.grey};
+                  /* border-bottom: 1px solid ${props =>
```

Comment:
Removing the border-bottom?

Reply:
Ah, it's at the other place now. I would remove this line, to make sure that it's parsed correctly. Styled-components sometimes has problems with commented template literals.

Reply:
That's actually a template string
```js
                css={`
                  position: absolute;
                  width: 100%;
                  background: #151515;
                  overflow: hidden;
                  /* border-bottom: 1px solid ${props =>
                    props.theme.homepage.grey}; */
                  
                  z-index: 99;
                  box-shadow: 0, 8px, 1rem rgba(0, 0, 0, 0.12), 0, 4px,
                    2px rgba(0, 0, 0, 0.24);
                  display: flex;
                  align-items: center;
                `}
```
, I'm quite sure that comment doesn't work as expected.